.github/ISSUE_TEMPLATE/1.bug.yml

```diff
@@ -0,0 +1,73 @@
+name: 🐛 Bug report
+description: File a bug report
+labels: ['status: unconfirmed', 'bug']
+projects: ['folkehelseinstituttet/12']
+body:
+  - type: markdown
+    attributes:
+      value: |
+        ## Thanks for taking the time to fill out this bug report!
+
+        Please fill out the sections below to help us reproduce and fix the bug you are encountering.
+
+        > [!TIP]
+        > We accept content written in norwegian if that is your preferred language.
+        > _Vi aksepterer innhold skrevet på norsk hvis det er ditt foretrukne språk._
+
+  - type: textarea
+    id: description
+    attributes:
+      label: Description
```

Comment:
description: Explain, in as much detail as possible, the bug you've met. Is this something that used to work and now doesn't?